cpp/proto/serializer.cc

```diff
@@ -443,44 +348,20 @@ SerializeResult TLSSerializer::WriteList(const repeated_string& in,
   size_t prefix_length = PrefixLength(max_total_length);
   CHECK_GE(length, prefix_length);
 
-  WriteUint(length - prefix_length, prefix_length);
+  WriteUint(length - prefix_length, prefix_length, output);
 
   for (int i = 0; i < in.size(); ++i)
-    WriteVarBytes(in.Get(i), max_elem_length);
-  return SerializeResult::OK;
-}
-
-SerializeResult TLSSerializer::WriteDigitallySigned(
-    const DigitallySigned& sig) {
-  SerializeResult res = CheckSignatureFormat(sig);
-  if (res != SerializeResult::OK)
-    return res;
-  WriteUint(sig.hash_algorithm(), Serializer::kHashAlgorithmLengthInBytes);
-  WriteUint(sig.sig_algorithm(), Serializer::kSigAlgorithmLengthInBytes);
-  WriteVarBytes(sig.signature(), Serializer::kMaxSignatureLength);
+    WriteVarBytes(in.Get(i), max_elem_length, output);
   return SerializeResult::OK;
 }
 
-
```

Comment:
Weird, should have left it alone:

https://github.com/google/certificate-transparency/blob/master/.clang-format#L2